--- a/Makefile.PL	2020-04-11 01:56:10.128628489 +0000
+++ b/Makefile.PL	2020-04-11 04:18:02.768428489 +0000
@@ -3,8 +3,8 @@
 use warnings;
 use Config;
 use ExtUtils::MakeMaker 6.64;
-use Alien::Librdkafka 0.9.3;
-use Alien::Base::Wrapper qw( Alien::Librdkafka !export );
+#use Alien::Librdkafka 0.9.3;
+#use Alien::Base::Wrapper qw( Alien::Librdkafka !export );
 use File::Spec ();
 
 WriteMakefile(
@@ -22,8 +22,8 @@
         'Test::More' => 0.94,
     },
     CONFIGURE_REQUIRES => {
-        'Alien::Base::Wrapper' => '0',
-        'Alien::Librdkafka' => '0.9.3',
+        #'Alien::Base::Wrapper' => '0',
+        #'Alien::Librdkafka' => '0.9.3',
         'ExtUtils::MakeMaker' => 6.64,
     },
     dist                => { COMPRESS => 'gzip -9f', SUFFIX => 'gz', },
@@ -39,9 +39,11 @@
         x_contributors => [
             'Pavel Shaydo <zwon@cpan.org>',
             'Graham Ollis <plicease@cpan.org>',
+            'Michael R. Davis <mrdvt@cpan.org>',
         ],
     },
-    Alien::Base::Wrapper->mm_args,
+    #Alien::Base::Wrapper->mm_args,
+    LIBS                => '-lrdkafka',
 );
 
 sub MY::postamble {
